mmv1/products/gkeonprem/VmwareCluster.yaml

```diff
@@ -0,0 +1,662 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: "VmwareCluster"
```

Comment:
Is there a published API reference that I can cross-reference?

Reply:
Here is the doc https://cloud.devsite.corp.google.com/anthos/clusters/docs/on-prem-api/reference/rest